src/ui/preferences/LinkSettings.qml

```diff
@@ -271,21 +275,21 @@ Rectangle {
                         height: ScreenTools.defaultFontPixelHeight * 0.5
                         width:  parent.width
                     }
-                    //-- Auto Connect on Start
-                    QGCCheckBox {
-                        text:       "Automatically Connect on Start"
-                        checked:    false
-                        visible:    editConfig ? editConfig.autoConnectAllowed : false
-                        onCheckedChanged: {
-                            if(editConfig) {
-                                editConfig.autoConnect = checked
-                            }
-                        }
-                        Component.onCompleted: {
-                            if(editConfig)
-                                checked = editConfig.autoConnect
-                        }
-                    }
+//                    //-- Auto Connect on Start
+//                    QGCCheckBox {
+//                        text:       "Automatically Connect on Start"
+//                        checked:    false
+//                        visible:    editConfig ? editConfig.autoConnectAllowed : false
+//                        onCheckedChanged: {
+//                            if(editConfig) {
+//                                editConfig.autoConnect = checked
+//                            }
+//                        }
+//                        Component.onCompleted: {
+//                            if(editConfig)
+//                                checked = editConfig.autoConnect
+//                        }
+//                    }
```

Comment:
Is this the auto connect feature that also works with the telemetry? I actually liked this... it can be disabled with the checkbox, right? So probably should not comment the source code out! :S Or does it interfere with other things?